packages/avatar/src/lib/Avatar.stories.tsx

```diff
@@ -0,0 +1,35 @@
+// Each exported component in the package should have its own stories file
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { Avatar, AvatarProps } from './Avatar';
+import { HomeIcon } from '@availity/mui-icon';
+
+const meta: Meta<typeof Avatar> = {
+  title: 'Components/Avatar/Avatar',
+  component: Avatar,
+  tags: ['autodocs'],
+};
+
+export default meta;
+
+export const _Avatar: StoryObj<typeof Avatar> = {
+  render: (args: AvatarProps) => <Avatar {...args} />,
+  args: {
+    children: 'This text is a child of Avatar',
```

Comment:
I think we should change this to match a use case in the portal. Would a first and last name showcase the children formatting better?

Reply:
updated